cordformation/src/main/kotlin/net/corda/plugins/External.kt

```diff
@@ -0,0 +1,28 @@
+package net.corda.plugins;
+
+import org.gradle.api.tasks.Input
+
+open class External {
+
+    @get:Input
+    var containerName: String = "external-service"
+
+    @get:Input
+    var servicePorts: List<Int> = listOf(8080)
```

Comment:
This becomes:
```kotin
 @get:Input
val servicePorts: ListProperty<Int> = objects.listProperty(Int::class.java).convention(listOf(8080))
```
Again, it doesn't need to be `@Optional`. However, I don't think a `ListProperty` needs to be `@Optional` anyway because its default value is an empty list.